samples/hello/hello.py

```diff
@@ -16,6 +16,11 @@
 
 
 def main() -> None:
+    """
+    an example showing how to create an synchronous connection to OpenSearch, create an index, index a document
+    and search to return the document
+    :return:
```

Comment:
Hello @macohen,

Thanks for your contribution! I've noticed that `:return: `appears to be empty at several places. Could you please fill it or remove it if not essential?

Reply:
done! thanks for the review. :)

Reply:
Linter is failing. Could you please run `nox -rs format`

Reply:
yeah. I saw that after I pushed. fixing log_collection_sample.py